srcCpp/Infrastructure_micro.cxx

```diff
@@ -28,6 +28,19 @@ PerftestClock::PerftestClock()
     OSAPI_NtpTime_from_millisec(&clockTimeAux, 0, 0);
     clockSec = 0;
     clockUsec = 0;
+
```

Comment:
You should not need the other implementation, right? No need to declare variables you are not using.

Reply:
Or initialize